pkg/csi/types/csi_types.go

```diff
@@ -59,22 +59,22 @@ func (c *CreatePVCArgs) Validate() error {
 type CreatePodArgs struct {
 	Name           string // Only one of Name or
 	GenerateName   string // GenerateName should be specified.
-	PVCName        string
+	PVCName        []string
```

Comment:
Hmm maybe this should be PVCMap
```suggestion
	PVCMap        map[string]VolumePath
```
```
type VolumePath struct {
    MountPath string
    DevicePath string
}
```

Reply:
Yeah. This actually makes more sense!

Reply:
Done ✅ 
We now use `PVCMap` everywhere instead if `PVCName`, `MountPath` & `DevicePath`